maps/aurora/aurora-1_centcomm.dmm

```diff
@@ -17967,7 +17967,9 @@
 /obj/item/weapon/tank/oxygen,
 /obj/item/clothing/shoes/magboots,
 /obj/item/clothing/shoes/magboots,
-/obj/item/weapon/rig/hazard/equipped,
+/obj/item/weapon/rig/hazard/equipped{
+	req_access = newlist()
```

Comment:
I think it should be null

Reply:
Those are the defaults of the industrial rig (which has no access restrictions)
![image](https://user-images.githubusercontent.com/1331699/56813983-6c8dbc80-683e-11e9-94be-810ac1944819.png)

Reply:
Huh, I thought they follow same principal as airlocks and machines which is null.